hphp/hack/hhi/stdlib/builtins_mysqli.hhi

```diff
@@ -0,0 +1,374 @@
+	public function num_rows (mysqli_stmt $stmt) {}
+	public function send_long_data ($param_nr, $data) {}
+	public function stmt () {}
+	public function free_result () {}
+	public function reset () {}
+	public function prepare ($query) {}
+	public function store_result () {}
+	public function get_result () {}
+}
+function mysqli_affected_rows ($link) {}
+function mysqli_autocommit ($link, $mode) {}
+function mysqli_begin_transaction ($link, $flags = 0, $name = null) {}
+function mysqli_change_user ($link, $user, $password, $database) {}
+function mysqli_character_set_name ($link) {}
+function mysqli_close ($link) {}
+function mysqli_commit ($link) {}
+function mysqli_connect ($host = '', $user = '', $password = '', $database = '', $port = '', $socket = '') {}
+function mysqli_connect_errno () {}
+function mysqli_connect_error () {}
+function mysqli_data_seek ($result, $offset) {}
+function mysqli_dump_debug_info ($link) {}
+function mysqli_debug ($message) {}
+function mysqli_errno ($link) {}
+function mysqli_error_list ($link) {}
+function mysqli_stmt_error_list ($stmt) {}
+function mysqli_error ($link) {}
+function mysqli_stmt_execute ($stmt) {}
+function mysqli_execute ($stmt) {}
+function mysqli_fetch_field ($result) {}
+function mysqli_fetch_fields ($result) {}
+function mysqli_fetch_field_direct ($result, $fieldnr) {}
+function mysqli_fetch_lengths ($result) {}
+function mysqli_fetch_all ($result, $resulttype = MYSQLI_NUM) {}
+function mysqli_fetch_array ($result, $resulttype = MYSQLI_BOTH) {}
+function mysqli_fetch_assoc ($result) {}
+function mysqli_fetch_object ($result, $class_name = '', $params = null) {}
+function mysqli_fetch_row ($result) {}
+function mysqli_field_count ($link) {}
+function mysqli_field_seek ($result, $fieldnr) {}
+function mysqli_field_tell ($result) {}
+function mysqli_free_result ($result) {}
+function mysqli_get_cache_stats ($link) {}
+function mysqli_get_connection_stats ($link) {}
+function mysqli_get_client_stats () {}
+function mysqli_get_charset ($link) {}
+function mysqli_get_client_info () {}
+function mysqli_get_client_version ($link) {}
+function mysqli_get_host_info ($link) {}
+function mysqli_get_proto_info ($link) {}
+function mysqli_get_server_info ($link) {}
+function mysqli_get_server_version ($link) {}
+function mysqli_get_warnings ($link) {}
+function mysqli_init () {}
+function mysqli_info ($link) {}
+function mysqli_insert_id ($link) {}
+function mysqli_kill ($link, $processid) {}
+function mysqli_set_local_infile_default ($link) {}
+function mysqli_set_local_infile_handler ($link, $read_func) {}
+function mysqli_more_results ($link) {}
+function mysqli_multi_query ($link, $query) {}
+function mysqli_next_result ($link) {}
+function mysqli_num_fields ($result) {}
+function mysqli_num_rows ($result) {}
+function mysqli_options ($link, $option, $value) {}
+function mysqli_ping ($link) {}
+function mysqli_poll (array &$read = null, array &$write = null, &$error = null, $sec, $usec = 0) {}
```

Comment:
Looks like we don't have that function, because that's mysqlnd. (Neither the method btw)

https://github.com/facebook/hhvm/blob/eab25645565957b29fafd7d9032c33d599b75df3/hphp/runtime/ext/mysql/ext_mysqli.php#L1956